spring-data-multi-storage-transaction-sample/src/main/java/sample/SampleService.java

```diff
@@ -0,0 +1,170 @@
+package sample;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.scalar.db.sql.springdata.EnableScalarDbRepositories;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.TransientDataAccessException;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import sample.domain.model.Customer;
+import sample.domain.model.Item;
+import sample.domain.model.ItemOrder;
+import sample.domain.model.Order;
+import sample.domain.model.OrderDetail;
+import sample.domain.model.Statement;
+import sample.domain.model.StatementDetail;
+import sample.domain.repository.CustomerRepository;
+import sample.domain.repository.ItemRepository;
+import sample.domain.repository.OrderRepository;
+import sample.domain.repository.StatementRepository;
+
+@EnableScalarDbRepositories
+@Service
+@Retryable(
+    include = TransientDataAccessException.class,
+    maxAttempts = 8,
+    backoff = @Backoff(delay = 1000, maxDelay = 8000, multiplier = 2))
+public class SampleService {
+  private final ObjectMapper objectMapper = new ObjectMapper();
+
+  @Autowired CustomerRepository customerRepository;
+  @Autowired ItemRepository itemRepository;
+  @Autowired OrderRepository orderRepository;
+  @Autowired StatementRepository statementRepository;
```

Comment:
Private scope is better for these cases?
```suggestion
  @Autowired private CustomerRepository customerRepository;
  @Autowired private ItemRepository itemRepository;
  @Autowired private OrderRepository orderRepository;
  @Autowired private StatementRepository statementRepository;
```

Reply:
Thanks! I tend to leave this kind of annotated fields non-private...

Reply:
I'll take care of other variables in `spring-data-sample` as well in another PR.